src/data/ACTIONS/index.js

```diff
@@ -30,7 +30,7 @@ import DRG from './DRG'
 const DEFAULT_GCD_CASTTIME = 0
 const DEFAULT_GCD_COOLDOWN = 2.5
 
-const ACTIONS = {
+const ACTIONS = addExtraIndex(addDefaultValues({
```

Comment:
I recall leylia having issues with intellisense not picking up the key names when `ACTIONS` was wrapped like this - does this still expose all the keys as before?

Reply:
![image](https://user-images.githubusercontent.com/73085/46872088-307ca680-ce6e-11e8-9820-d679e3883301.png)